lynx/common/actions/chop.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+
+from lynx.common.actions.action import Action
+from lynx.common.vector import Vector
+from lynx.common.actions.removeObject import RemoveObject
+from lynx.common.actions.createObject import CreateObject
+from lynx.common.object import Object
+from lynx.common.actions.common_requirements import CommonRequirements
+
+@dataclass
+class Chop(Action):
+	"""
+	Simple action used to hit/destroy object, which stands on the given destination.
+	"""
+	object_id: int = -1
+	position: Vector = Vector(1, 0)
+
+	def apply(self, scene: 'Scene') -> None:
+		objects_on_square = scene.get_objects_by_position(self.position)
+		for object in objects_on_square:
+			if object.name == 'tree':

```

Comment:
We can also create a static class with object names instead of using 'tree' as a literal value.
e.g.
```
class ObjectNames:
    TREE = 'tree'
    GRASS = 'grass'
    AGENT = 'agent'
    LOG = 'log'
```